gencode.c

```diff
@@ -10715,3 +10715,55 @@ gen_atmmulti_abbrev(compiler_state_t *cstate, int type)
 	}
 	return b1;
 }
+
+/*
+ * support IEEE 802.1Q VLAN trunk over ethernet
```

Comment:
This needs to refer to the most appropriate specification and/or to give a packet header diagram, otherwise it is difficult to reason whether the code is correct.

Reply:
Thanks! I missed that on a copy/paste. Since 802.1Qbh was never formally accepted, this is still a proprietary extension used by some vendors. 802.1Qbr got accepted, but is a little different with a different EtherType. So, I've added a packet layout as well.